docs/index.md

```diff
@@ -75,37 +82,38 @@ Directives customize how values are handled, allowing for asynchronous values, e
 
 <section>
 <div class="wrapper">
-<div class="responsive-row center">
-<div style="max-width: 600px">
 
+<h1 class="title">Efficiently creating and updating DOM</h1>
+<h3 style="max-width: 560px">
 lit-html is not a framework, nor does it include a component model. It focuses on one thing and one thing only: efficiently creating and updating DOM. It can be used standalone for simple tasks, or combined with a framework or component model, like Web Components, for a full-featured UI development platform.
+</h3>
 
-</div>
-</div>
 </div>
 </section>
 
 <section>
-<div class="wrapper" style="text-align: center">
-<h2>Browser Compatibility</h2>
-<p>lit-html works in all major browsers (Chrome, Firefox, IE, Edge, Safari, and Opera). </p>
-<div>
-<img width="70" height="70" src="/images/browsers/chrome_128x128.png" alt="chrome logo">
-<img width="70" height="70" src="/images/browsers/firefox_128x128.png" alt="firefox logo">
-<img width="70" height="70" src="/images/browsers/internet-explorer_128x128.png" alt="internet explorer logo">
-<img width="70" height="70" src="/images/browsers/edge_128x128.png" alt="edge logo">
-<img width="70" height="70" src="/images/browsers/safari_128x128.png" alt="safari logo">
-<img width="70" height="70" src="/images/browsers/opera_128x128.png" alt="opera logo">
+<div class="wrapper">
+
+<h1 class="title">Browser Compatibility</h1>
+<h2 class="description">lit-html works in all major browsers (Chrome, Firefox, IE, Edge, Safari, and Opera). </h2>
+<div id="browser-thumbnails" style="margin-bottom: 20px;">
+<img width="56" width="56" src="{{ site.baseurl }}/images/browsers/chrome_128x128.png" alt="chrome logo">
```

Comment:
Should the browser names be capitalized in the `alt`?

Reply:
Updated alt label accordingly.